vscode/package.json

```diff
@@ -125,7 +131,25 @@
           }
         }
       }
-    ]
+    ],
+    "configuration": {
+      "title": "Q#",
```

Comment:
Reviewer: If you're going to nitpick on the documentation here, please also nitpick on the documentation for the Python library as it's lifted directly from there.

https://github.com/microsoft/qsharp/blob/f2529e17d5ff21b189e846d456702ea077e989b8/pip/qsharp/_native.pyi#L7